draft-ietf-tls-tls13.md

```diff
@@ -1370,6 +1370,13 @@ ClientHello (without modification) except:
 If a server receives a ClientHello at any other time, it MUST send
 a fatal "unexpected_message" alert and close the connection.
 
+If a server receives a TLS v1.3 ClientHello inside already established TLS
+session (as would be the case of the renegotiation mechanism from previous
+versions of TLS protocol), it MUST NOT negotiate TLS v1.3 but continue
```

Comment:
"inside already established" sounds like it might apply to tunneling a ClientHello over the application stream. "would" is also a bit string. I don't think the spec says you have to keep ClientHello.version fixed on renego. It's just that everyone does in practice due to past interop issues.

Perhaps:
If a server has established a TLS session at a previous version of the protocol and receives a TLS 1.3 ClientHello in a renegotiation, it MUST retain the previous protocol version. In particular, it MUST NOT negotiate TLS 1.3.